configs/README.md

```diff
@@ -1,15 +1,23 @@
 ## Configuration File
-Example configuration files are provided in this directory. To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md). A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below. The asterisk (*) denotes calibratable parameters.
+Example configuration files are provided in this directory.
+To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md).
+A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below.
+> [!NOTE]
+> The asterisk (*) denotes calibratable parameters.
+> Note, calibratable parameters are passed via BMI using the _name next to the asterisk_.
+>
+> Slashes (/) in the variable column denote that a parameter can be specified using either name.
+> For example, `soil_params.b` can equivalently be provided as `b`.
 
 | Variable | Datatype |  Limits  | Units | Role | Process | Description |
 | -------- | -------- | ------ | ----- | ---- | ------- | ----------- |
 | forcing_file | *char* | 256  |   | filename |   | path to forcing inputs csv; set to `BMI` if passed via `bmi.set_value*()`  |
 | soil_params.depth | *double* |   | meters [m]| state |  | soil depth  |
-| *soil_params.b | *double* |   |   | state |   | beta exponent on Clapp-Hornberger (1978) soil water relations  |
-| *soil_params.satdk | *double* |   |  meters/second [m s-1] | state |  | saturated hydraulic conductivity  |
-| *soil_params.satpsi  | *double* |   |  meters [m] | state |  | saturated capillary head  |
-| *soil_params.slop   | *double* |   |  meters/meters [m/m]| state |  | this factor (0-1) modifies the gradient of the hydraulic head at the soil bottom.  0=no-flow. |
-| *soil_params.smcmax  | *double* |   |  meters/meters [m/m] | state |  | saturated soil moisture content  |
+| soil_params.b / *b | *double* |   |   | state |   | beta exponent on Clapp-Hornberger (1978) soil water relations  |
+| soil_params.satdk / *satdk | *double* |   |  meters/second [m s-1] | state |  | saturated hydraulic conductivity  |
+| soil_params.satpsi / *satpsi | *double* |   |  meters [m] | state |  | saturated capillary head  |
+| soil_params.slop / *slope | *double* |   |  meters/meters [m/m]| state |  | this factor (0-1) modifies the gradient of the hydraulic head at the soil bottom.  0=no-flow. |
+| soil_params.smcmax / *maxsmc | *double* |   |  meters/meters [m/m] | state |  | saturated soil moisture content  |
```

Comment:
instead of `/` I would suggest using `soil_params.b or *b`. We use `/` for division `[m/m]`